gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/util/DefaultGraphManager.java

```diff
@@ -42,8 +45,8 @@
  * the configuration file. The {@link TraversalSource} instances are rebound to the {@code GraphManager} once
  * initialization scripts construct them.
  */
-public final class DefaultGraphManager implements GraphManager {
-    private static final Logger logger = LoggerFactory.getLogger(GremlinServer.class);
```

Comment:
I think you should keep `GremlinServer.class` for the logger name (and prefer it as well for the `CheckedGraphManager`. We've considered the `GraphManager` part of the general Gremlin Server startup. I don't think there's much value in being able to configure the `GraphManager` logs separate from `GremlinServer`.

Reply:
I have changed not for configuration purpose, but to easily be able to find the source code from a log entry.  
It has been hard for me to debug, as I saw a log with GremlinServer, but I could not find the log in the source code for this class.
Using a logger per class does not make configuration harder, as configuration can be set at package level (for instance `org.apache.tinkerpop.gremlin.server: INFO`)

Reply:
true - i withdraw that suggestion.